### ISSUE TITLE
Ability to Order Submodels in Sequencer

### ISSUE THREAD

**Issue by MrPaulAR** [smeighan/xLights#2672]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
The ability to order submodels within the sequencer.  A specific use case is when using something like the Grand Illusion or Mesmerizer where there are many submodels.   Now that I've starting moving from groups of single submodels to 1 submodel with layers the ordering is static inside the sequencer.  

This should not be a breaking change.  If the display elements does not have this option prior to upgrade then carry over the existing ordering.  I thought it was alphabetical but my display elements is not so perhaps it carries over from the layout.

**Describe alternatives you've considered**
I can still make a group with the submodel but it's a group with 1 item in this case and then use the display elements to reorder them as indicated below.
* GROUP Grand Illusion/Spinner
* GROUP Grand Illusion/Rings

**Additional context**
None

**Comment by sslupsky** [comment]
I agree with this.  It issue is quickly becoming a big mess.  Every sequence vendor has their specific way of organizing their layout.  Some vendors use the submodels as they are defined in the model and others create groups with one submodel as described above.  If you use sequences from several vendors, the number of model groups required in your layout quickly explodes and it becomes difficult to track which model groups come from which vendor.  This problem creates a dependency on layout.  @MrPaulAR request to order submodels using display elements in is a good solution.

**Comment by cybercop23** [comment]
There's a new feature in 2016.10 - right-click option to hide unused submodels when expanding a model row
Not exactly the same as the ask, but will this work? Rearranging submodels has rendering implications and *now* (many years later) vendors number their submodels
